### ISSUE TITLE
Heads Up: Binder use at MIT for one assignment this January

### ISSUE THREAD

**Issue by shvonski** [jupyterhub/team-compass#250]
Hi All,

I forgot to mention that we want to try out ONE use case of mybinder.org in a Physics class this January. We were planning to give it to students as part of an optional assignment due the week of 1/20/20. We wanted to observe how easy it was for students to follow/initiate/use. Will also try to give survey.
 
The class has ~70 students. The repo is: https://github.com/shvonski/test3c
 
Just wanted to let you know!

**Comment by choldgraf** [comment]
Yeah! Go for it! :-)

**Comment by sgibson91** [comment]
Did this happen? How did you find the experience?

**Comment by shvonski** [comment]
Yes, we did this without issue (well, one slight issue, see below). We used
it in two ways, and asked students to fill out a survey. Some results below:

1) We explicitly used it in one recitation with ~15 people. Everyone was
able to load the notebook and interact with the assignment (load times
<2min), except one user. Her notebook *never loaded* (even after about 10
min). She was using Chrome (but others used Chrome without issue). I don't
recall that there were any error messages, simply a black terminal. From
this group, there were 11 respondents to our survey.

2) Although it was emailed to the entire 70 person class as an optional
assignment, very few took advantage. Besides those who explicitly used it
in recitation, only 4 additional students responded to our survey.

General takeaways: This was easy enough for students to do, but very few
took advantage of it.
